docs/Other SDL Features/Calling a Phone Number/index.md

```diff
@@ -1,58 +1,75 @@
+## Checking Your App's Permissions
+@![iOS]`SDLDialNumber`!@@![android,javaSE,javaEE]`DialNumber`!@ is an RPC that is usually restricted by OEMs. As a result, a module may reject your request if your app does not have the correct permissions. Your SDL app may also be restricted to only being allowed to making a phone call when your app is open (i.e. the `hmiLevel` is non-`NONE`) or when it is the currently active app (i.e. the `hmiLevel` is `FULL`). 
 
 @![iOS]
 ##### Objective-C
 ```objc
-[self.sdlManager.systemCapabilityManager updateCapabilityType:SDLSystemCapabilityTypePhoneCall completionHandler:^(NSError * _Nullable error, SDLSystemCapabilityManager * _Nonnull systemCapabilityManager) {
-    BOOL isDialNumberSupported = NO;
-    if (error == nil) {
-        isDialNumberSupported = systemCapabilityManager.phoneCapability.dialNumberEnabled.boolValue;
-    }
-    else {
-        isDialNumberSupported = systemCapabilityManager.hmiCapabilities.phoneCall.boolValue;
+// TODO
+```
+
+##### Swift
+```swift
+_  = sdlManager.permissionManager.addObserver(forRPCs: [SDLRPCFunctionName.dialNumber.rawValue.rawValue], groupType:.any, withHandler: { (individualStatuses, groupStatus) in
+    guard groupStatus == SDLPermissionGroupStatus.allowed else { 
+        // Your app does not have permission to send the `DialNumber` request for its current HMI level
+        return 
     }
 
-    <#If making phone calls is supported, send the `DialNumber` RPC#>
-}];
+    // Your app has permission to send the `DialNumber` request for its current HMI level
+})
+```
+!@
+
+@![android,javaSE,javaEE]
+```java
+// TODO
+```
+!@
+
+## Checking if the Module Supports Calling a Phone Number
+Since making a phone call is a newer feature, there is a possibility that some legacy modules will not support your request because the module does not support the @![iOS]`SDLDialNumber`!@@![android,javaSE,javaEE]`DialNumber`!@ request. Once you have successfully connected to the module, you can check the module's capabilities via the @![iOS]`SDLManager.systemCapabilityManager`!@@![android, javaSE, javaEE]`sdlManager.getSystemCapabilityManager`!@ as shown below:
+
+@![iOS]
+##### Objective-C
+```objc
+// TODO
 ```
 
 ##### Swift
 ```swift
-sdlManager.systemCapabilityManager.updateCapabilityType(.phoneCall) { (error, systemCapabilityManager) in
-    var isDialNumberSupported = false
-    if error == nil {
-        isDialNumberSupported = systemCapabilityManager.phoneCapability?.dialNumberEnabled?.boolValue ?? false;
+// Check if the module supports making a phone call
+guard (sdlManager.systemCapabilityManager.isCapabilitySupported(type: .phoneCall)) else { return }
```

Comment:
Turn this into a method to check if `DialNumber` is supported, or something like that. You don't want to call `updateCapabilityType` _every_ time you are going to go to call `DialNumber` so the value should probably be cached. Maybe make a small class to handle checking and sending `DialNumber`? Or a method? Also, tell the dev to disable any buttons that would send `DialNumber` if not supported / the app doesn't have permissions.

Reply:
Added a method and a paragraph telling devs to disable the buttons